nicoRDL/dotTheory-FEv2
# FE-401: Assembling Sass

## Description
- refactors css files to scss files, converting to Sass
- refactors view files to sass convertions in styles

## Related Ticket
https://codeandtheory.atlassian.net/browse/FE-654

## Testing Instructions
1. Pull on branch FE-401/FE-654
2. Evaluate the use of Sass in the stylesheets
3. Confirm that website and all web pages are showing correct styles from previous css implementation

## RIsks
N/A
